composer.json

```diff
@@ -5,7 +5,7 @@
     "require": {
         "nette/finder": "^2.4.0",
         "doctrine/annotations": "^1.13.0",
-        "doctrine/dbal": "^2.13.1",
+        "doctrine/dbal": "^2.13.1|^3.5.1",
```

Comment:
Is this the lowest possible with major 3?

Reply:
The downgrade to 3.3.1 is done.